tests/test_api.py

```diff
@@ -106,6 +113,10 @@ def __init__(self, z):
         self.assertEqual(recorded_reads, {})
         self.assertEqual(recorded_writes, {})
 
+        map_callback.assert_not_called()
+        z.memory.map(0xB0BAF000, 0x1000)
+        map_callback.assert_called_with(z, 0, 0xB0BAF000, 0x1000, None)
```

Comment:
If you change the above & hooks are called with data (not `None`), update the test here.

Reply:
Caused some of our LTP tests to fail due to timeouts. Left some comments stating that we won't pass the "value" part of the memory callback due to time costs.